x/rollapp/types/rollapp.go

```diff
@@ -68,8 +69,10 @@ func (r Rollapp) ValidateBasic() error {
 		return errorsmod.Wrap(ErrInvalidInitialSequencer, err.Error())
 	}
 
-	if err = validateBech32Prefix(r.Bech32Prefix); err != nil {
-		return errorsmod.Wrap(ErrInvalidBech32Prefix, err.Error())
+	if r.Sealed || r.Bech32Prefix != "" {
```

Comment:
not sure I understand why we check for if sealed here? 

this validate basic is called when:
1. rollapp creation (don't need to validate sealed cause for sure it's not on creation)
2. rollapp update (if we try to update bech32, it should fail before in the `UpdatingImmutableValues() && current.Sealed`

Reply:
I guess I was being too strict (for import genesis)

I'll remove